src/POData/SimpleDataService.php

```diff
@@ -73,7 +73,7 @@ public function getMetadataProvider()
      */
     public function getStreamProviderX()
     {
-        // TODO: Implement getStreamProviderX() method.
+        return new POData\Providers\StreamSimpleStreamProvider();
```

Comment:
1 - typo/missing separator?
2 - Wouldn't it be easier to inject this into constructor, so we can reuse same stream provider rather than a new one for each and every call?